src/main/webapp/app/exam/manage/exams/exam-detail.component.html

```diff
@@ -109,17 +109,17 @@ <h3>Exam Details:</h3>
                     <ng-template #moduleNumberNotSet><span jhiTranslate="artemisApp.exam.notSet"></span></ng-template>
                 </dd>
                 <dt><span jhiTranslate="artemisApp.examManagement.visibleDate">Visible from</span></dt>
-                <dd>
+                <dd id="exam-visible-date">
```

Comment:
Maybe we could prefix ids that are relevant for javascript or tests with
`js-` or `test-` e.g. `js-exam-visible-date`

Reply:
Cypress actually has an opinion here that should be followed I think
https://docs.cypress.io/guides/references/best-practices#Selecting-Elements